bower.json

```diff
@@ -23,6 +23,7 @@
     "angular-ui-grid": "3.0.7",
     "angular-ui-validate": "1.2.2",
     "bootstrap": "3.3.6",
+    "lodash": "4.5.0",
```

Comment:
I'd argue yes, simply because it provides functionality we should not try to re-implement ourselves. Also, I think this will allow us to clean up some of the existing code. But good point on the version though.